src/sentry/api/serializers/models/sentry_app.py

```diff
@@ -91,13 +94,14 @@ def serialize(self, obj, attrs, user, access):
                 is_active_superuser(env.request) or is_active_staff(env.request)
             )
             if elevated_user or owner.id in user_org_ids:
+                has_secret = obj.date_added > timezone.now() - timedelta(days=1)
```

Comment:
nit: I think `show_secret` might be a better name for this, because that's what this is helping determine and holding the value of right?